standalone/main.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2020 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <chrono>
+#include <sstream>
+#include <string>
+#include <thread>
+
+#include <psen_scan_v2_standalone/core.h>
+#include <psen_scan_v2_standalone/logging.h>
```

Comment:
Maybe combine logging also into the helper include?
```suggestion
#include <psen_scan_v2_standalone/core.h>
```

Reply:
Done